sdk/digitaltwins/Azure.DigitalTwins.Core/src/DigitalTwinsClient.cs

```diff
@@ -867,12 +881,12 @@ public virtual Task<Response<string>> CreateRelationshipAsync(string digitalTwin
         /// <exception cref="ArgumentNullException">
         /// The exception is thrown when <paramref name="digitalTwinId"/> or <paramref name="relationshipId"/> is <c>null</c>.
         /// </exception>
-        /// <seealso cref="CreateRelationshipAsync(string, string, string, CancellationToken)">
+        /// <seealso cref="CreateRelationshipAsync(string, string, string, DigitalTwinsAddRelationshipOptions, CancellationToken)">
         /// See the asynchronous version of this method for examples.
         /// </seealso>
-        public virtual Response<string> CreateRelationship(string digitalTwinId, string relationshipId, string relationship, CancellationToken cancellationToken = default)
+        public virtual Response<string> CreateRelationship(string digitalTwinId, string relationshipId, string relationship, DigitalTwinsAddRelationshipOptions options = null, CancellationToken cancellationToken = default)
```

Comment:
Same here, consider changing this option name from `DigitalTwinsAddRelationshipOptions` -> `DigitalTwinsCreateRelationshipOptions`
Basically everywhere we do Add or List in the option name, I think we should rename to Create and Get